src/plural/general.ts

```diff
@@ -0,0 +1,8 @@
+export function pluralizer (count: number, lang: string): Intl.LDMLPluralRule {
+    const pluralCategories = new Intl.PluralRules(lang).resolvedOptions().pluralCategories;
```

Comment:
Do all browsers support `Intl.PluralRules`? And if the library will be used on the server NodeJS?
Need to add a check that `Intl.PluralRules` is defined.

```
 if (!Intl.PluralRules) {
    throw Error('Intl.PluralRules is not available. Use polyfill.');
  }
```

Reply:
All last browsers and NodeJS support Intl.PluralRules 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/PluralRules#browser_compatibility